src/persist-client/benches/benches.rs

```diff
@@ -36,23 +43,54 @@ pub fn bench_persist(c: &mut Criterion) {
         .expect("Failed building the Runtime");
     let runtime = Arc::new(runtime);
 
-    // Intentionally use the small generator for min_latency benches, because
-    // y'know... min latencies. :)
-    let data_small = DataGenerator::small();
-
-    min_latency::bench_writes(
-        &mut c.benchmark_group("min_latency/writes"),
-        Arc::clone(&runtime),
-        &data_small,
-    )
-    .expect("failed to bench writes");
+    // Default to latency. First, because it's usually more interesting for
+    // these micro-benchmarks (throughput is more the domain of the open-loop
+    // benchmark). Second, because this allows `cargo test --all-targets` to
+    // finish in a reasonable time for this crate, even without --release.
+    //
+    // However, also allow an easy toggle for a quick, rough estimate of
+    // throughput without going through the involvement of the open-loop
+    // benchmark.
+    let throughput = mz_ore::env::is_var_truthy("MZ_PERSIST_BENCH_THROUGHPUT");
+    let data = if throughput {
+        eprint!("MZ_PERSIST_BENCH_THROUGHPUT=1 ");
+        DataGenerator::default()
+    } else {
+        DataGenerator::small()
+    };
 
+    min_latency::bench_writes("min_latency/writes", throughput, c, &runtime, &data);
```

Comment:
One of my motivations for the new structure was that the old persist benchmarks had started to become a bit of a jumble. Concretely, there were a number of them that only existed to help tune a specific PR and then stuck around, adding cruft everything we needed to benchmarks to evaluate a change. Here, I'm trying to break them up first roughly into 2 categories:
- "speed_of_light" (accepting name suggestions) which is a measure of fundamental things that persist's latency and throughput are dependent on. This basically means BlobMulti and Consensus impls, though I also tossed batch encoding in here. The latter is a bit different (it's indeed something we control) but it's also a sort of fundamentally separate thing from the rest of persist, especially with aljoscha's PR to spearate out a BatchBuilder in our interface.
- Then a group of things that are roughly "what our users see". They're called "min_latency" because that's what they were called before this PR and I haven't come up with any names that I like for this (again accepting suggestions).

I'm hoping that going forward, we can start establishing easily targeted (on the cargo bench cli) subsets of these for common situations. Walking through a couple examples:
- A PR _might_ affect latencies: `min_latency/*/mem_mem` will tell whether this makes a measurable impact at all. If it does, then as a followup `min_latency/*/s3_pg` tells whether it shows up in prod and `min_latency/*/file_pg` ditto for dev.
- We want to get a sense for persist's current prod latencies (say to update the README): `min_latency/*/s3_pg`
- We want to figure out how much overhead persist gives over the fundamental pieces we have no control over: `min_latency/writes/s3_pg` vs `speed_of_light/blob_set/s3` + `speed_of_light/consensus_compare_and_set/pg`
- We want a sense for what our max throughput is for open-loop: MZ_PERSIST_BENCH_THROUGHPUT=1 + `speed_of_light/blob_set/s3`
- Evaluating perf work within an individual BlobMulti or Consensus impl: `speed_of_light/*/<foo>`

One takeaway I'm getting from writing this comment up is that I dunno if `cargo test` supports the `*` wildcard, so we probably want to move the mem, file, s3, sqlite, pg stuff to be the second slot, right after `min_latency` or `speed_of_light`. I'll hold off until we get some PR comments here first, so I can make a batch of changes at once